lbrynet/core/Wallet.py

```diff
@@ -609,7 +609,21 @@ def update_metadata(self, new_metadata, old_metadata):
             meta_for_return[k] = new_metadata[k]
         return defer.succeed(Metadata(meta_for_return))
 
+    """
+    Claim a name, update if name already claimed by user
+    @param name: name to claim
 
+    @param bid: bid amount
+
+    @param m: metadata
+
+    @return: Deferred which returns a dict containing below items
+        txid - txid of the resulting transaction
+        nout - nout of the resulting claim
+        fee - transaction fee paid to make claim
+        claim_id -  claim id of the claim
+
+    """
```

Comment:
Other functions in Wallet.py does not use the google style. I'll fix the commenting style of this and other function in Wallet.py  in a seperate pull request.